### PR TITLE
Switch to standardjs

### PR DESCRIPTION
**Pull request by gabrielmontagne**
## Description

Switch to [standardjs](http://standardjs.com/) for format and use it for linting.
Lots of silly things (sont variables, etc.) cleaned up along the way.
No functional or API changes at all.

Basically ran `standard --fix` until it came out empty.

## Motivation and Context

From @mamapitufo I learned how to stop worrying about quirky code layouts and love the bomb.
In the quest for minimizing the area of code and configuration one needs to tend, this single, pre-configured setup should be the way to go!  d3-utils and grid-components should get the treatment soon.

## How Was This Tested?

Running all the included examples in the `examples` folder.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My change follows the style of this project
- [ ] My change requires a change to the documentation
- [ ] I have updated the documentation accordingly
- [x] I have read the [contribution guidelines](../CONTRIBUTING.md)
- [ ] I have added tests to cover my changes
- [ ] All new and existing tests passed

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@zambezi/grid",
-  "version": "0.18.3-selection-changed-leak-fix.1",
+  "version": "3.10.4-switch-to-standardjs.1",
```
Comment (mstade) [inline package.json]: Quite the version bump. :o)

Comment (gabrielmontagne) [inline package.json]: wow, I wonder where that came from!  let me fix

**Comment by mamapitufo** [comment]
I'm afraid I don't have much to say on the matter, just that: having a coding standard is good (you don't waste time arguing on whitespace); standardjs doesn't have many "offensive" settings (in my particular case the only thing I had to change was the space between the function name and args list); this one is easy to configure, and it has a `--fix` option that facilitates the initial migration.

I'm sure there are other coding standards that are as good or bad as this one, some of them might even have great tooling... I just happened to find this one first, via @cristiano-belloni :)

File: package.json (hunk 1 above)
Comment (mamapitufo) [inline package.json]: at first I thought it was some sort of pi-day release...

**Comment by cristiano-belloni** [comment]
Pretty much what @mamapitufo said. It's handy to agree on a standard linting that's reasonable, widely used and is maintained by not-us + it's actually useful to catch various basic code smells (like unused variables) that hide dead code or typos. I used airbnb's linting at Sky and I hated it: incredibly invasive, used semicolons and was a pain to configure. Standard is pretty much what we normally write,  with an additional space between function declaration and params, and with postfix comma. It's easy to setup, you don't need a huge rc file (actually you don't need it at all if you use the provided cli) and it has an active community, so it's well-maintained.

**Comment by gabrielmontagne** [comment]
One can never quote enough [Fredkin](https://en.wikipedia.org/wiki/Edward_Fredkin)'s paradox,
> The more equally attractive two alternatives seem, the harder it can be to choose between them—no matter that, to the same degree, the choice can only matter less

Plus zero configuration.

**Pull request merged**